corehq/apps/userreports/pillow.py

```diff
@@ -181,7 +181,7 @@ def rebuild_table(self, adapter):
             latest_rev = config.get_db().get_rev(config._id)
             if config._rev != latest_rev:
                 raise StaleRebuildError('Tried to rebuild a stale table ({})! Ignoring...'.format(config))
-        adapter.rebuild_table()
+            adapter.rebuild_table()
         if config.is_static:
```

Comment:
Might make sense to change this to `else:`

Reply:
```suggestion
        else:
```